.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+indent_size = 2
```

Comment:
Shouldn't this be more something like this?
```
indent_style = tab
indent_size = tab
tab_width = 4
```
(At least for c++ and lua files (`[*.{cpp,lua,c,h}]`).)

Reply:
This is for the website files, which all use 2-space indentation :slightly_smiling_face:

That said, I could also open a PR for the Minetest and minetest_game repositories.

Reply:
Oops, sorry, I didn't look at the repository name.